examples/generalPermissionManager.ts

```diff
@@ -129,27 +128,35 @@ window.addEventListener('load', async () => {
   await generalPM.addDelegate({ delegate: myAddress, details: 'details' });
 
   const generalTMAddress = (await tickerSecurityTokenInstance.getModulesByName({
-    moduleName: ModuleName.generalTransferManager,
+    moduleName: ModuleName.GeneralTransferManager,
   }))[0];
 
   // Get all delegates
-  await generalPM.changePermission({ valid: true, perm: 'FLAGS', delegate: myAddress, module: generalTMAddress });
+  await generalPM.changePermission({ valid: true, perm: 'ADMIN', delegate: myAddress, module: generalTMAddress });
 
   // Check  delegate
   console.log('Delegate is added:');
   console.log(await generalPM.checkDelegate({ delegate: myAddress }));
   console.log('Delegate has flags perm added on general transfer manager:');
   console.log(await generalPM.checkPermission({ delegate: myAddress, module: generalTMAddress, permission: 'FLAGS' }));
 
+
+  // Change allow all whitelist transfers: I_GeneralTransferManager.modifyTransferRequirementsMulti(
+  //                 [0, 1, 2],
+  //                 [false, false, false],
+  //                 [false, false, false],
+  //                 [false, false, false],
+  //                 [false, false, false],
+  //                 { from: token_owner }
   // Use FLAGS permission to allow all whitelist transfers, this validates that the user can use the
   const generalTM = await polymathAPI.moduleFactory.getModuleInstance({
-    name: ModuleName.generalTransferManager,
+    name: ModuleName.GeneralTransferManager,
     address: generalTMAddress,
   });
   await generalTM.changeAllowAllWhitelistTransfers({ allowAllWhitelistTransfers: true });
 
   // Revoking Permission
-  await generalPM.changePermission({ valid: false, perm: 'FLAGS', delegate: myAddress, module: generalTMAddress });
+  await generalPM.changePermission({ valid: false, perm: 'ADMIN', delegate: myAddress, module: generalTMAddress });
 
   console.log('Delegate perm has been revoked. Check permission result: ');
   console.log(await generalPM.checkPermission({ delegate: myAddress, module: generalTMAddress, permission: 'FLAGS' }));
```

Comment:
These console.log statements should change to reflect the change in permission used